app/css/Affirmation.css

```diff
@@ -77,18 +76,30 @@
 }
 
 .carousel-button {
-  color: 'black';
+  color: "black";
   background: transparent;
 }
 
 .carousel-button-disabled {
-  color: 'gray';
+  color: "gray";
   background: #eaecef;
 }
 
-.carousel-button-disabled, .carousel-button-disabled:hover {
+.carousel-button-disabled,
+.carousel-button-disabled:hover {
   background-color: #eaecef; /* Disabled state background */
   pointer-events: none;
   opacity: 0.5;
 }
 
+.css-1orgm40 {
+  padding: 0px !important;
+}
+
+.-left-12 {
+  margin-left: 60px !important;
+}
+
+.-right-12 {
+  margin-right: 60px !important;
```

Comment:
@XiaoStevens, what are those css classes for?

Reply:
![affirmation carousel](https://github.com/user-attachments/assets/5b72689d-f444-485f-aba2-3de8fdeafc72)
1. remove the gap between the carousel and window edges.
2. make arrows showing on the carousel panel.

Reply:
> @XiaoStevens, what are those css classes for?

Those are for adjusting arrows' position. 
1. Remove padding for both arrows.
2,3. Adding margin to both left arrow and right arrow.